Data/translations/Covid Localization - Home Page.csv

```diff
@@ -27,21 +35,31 @@ Data to {{1}}
 
 ### Current
 
-{{2}} **{{3}}** days
+{{2}} **{{3}}** days (based on health care worker absenteeism due to COVID-19)

```

Comment:
Could you delete `(based on health care worker absenteeism due to COVID-19)` and replace with `*`.

Also on line 40.

Then after line 44 put a new line:
`* Based on health care worker absenteeism due to COVID-19.`